doc/articles/dummies1.md

```diff
@@ -0,0 +1,138 @@
+# How to migrate existing UWP code to Uno
+
+There are two separate paths for using an existing UWP codebase on top of uno:
+- An existing UWP application
+- An existing UWP library
+
+For migrating UWP library, see [this article](howto-migrate-existing-code.md).
+
+# How to migrate UWP app to Uno?
+# Step by step guide.
+
+This guide assumes that you have some UWP app, and want to convert this app to be multiplatform.
+
+ Maybe your programmers experience doesn't include Git, Android, iOS, etc. Maybe you know only UWP. And maybe even your apps are written in VB... But still, you can not only use Uno, but also contribute to it. Please, read this short guide, guide for beginners (or dummies...).
```

Comment:
Delete this passage, it's not really necessary.

Reply:
ok